reference/catalog.v3.yml

```diff
@@ -409,6 +409,16 @@ paths:
             type: array
             items:
               type: string
+        - schema:
+            type: string
+          in: query
+          name: channel_id
```

Comment:
Can you please check the description of this ticket and the implementation https://bigcommercecloud.atlassian.net/browse/MSF-2510 
as it was updated to be an array of `channels`

Reply:
@kristinapototska please re-review